modules/building/pages/konflux-build-binaries.adoc

```diff
@@ -0,0 +1,19 @@
+= Building Binaries in {ProductName}
+
+This guide provides guidelines on how to build binaries within {ProductName}.
+
+== Theory
+
+The fundamental process involves creating a Dockerfile to encapsulate your binary build process. This is similar to your existing build process, but it's executed inside a Docker container. You'll then define your application and component in {ProductName} as you normally would. For details on this, refer to xref:creating.adoc[Creating applications and components].
+
+If you're unfamiliar with Dockerfiles, consult the official link:https://docs.docker.com/reference/dockerfile/[documentation].
```

Comment:
We should use `Containerfiles` but there isn't as clear of an authoritative documentation source as there is for a Dockerfile.

https://github.com/containers/buildah/blob/main/docs/buildah-build.1.md ?

Reply:
What do you suggest then? I can change it to Container file, but I still prefer the Dockerfile reference docs. Maybe I can remove the link altogether

Reply:
Fixed